src/index.js

```diff
@@ -1,10 +1,92 @@
 #!/usr/bin/env node
 
-require('yargs')
+/**
+ * Networks available
+ */
+let networks = {
+  mainnet: 'https://mainnet.infura.io',
+  ropsten: 'https://ropsten.infura.io',
+  rinkeby: 'https://rinkeby.infura.io',
+  kovan: 'https://kovan.infura.io',
+  sokol: 'https://sokol.poa.network',
+  poa: 'https://core.poa.network',
+  local: 'http://localhost:8545'
+}
+
+let yargs = require('yargs')
   .option('url', {
     description: 'URL of the ethereum node to connect',
-    default: 'http://localhost:8545'
+    default: 'http://localhost:8545',
+    type: 'string'
+  })
+  .option('mainnet', {
+    describe: `Url of the mainnet ethereum node to connect: ${networks.mainnet}`,
+    type: 'boolean'
+  })
+  .option('ropsten', {
+    describe: `Url of the ropsten ethereum node to connect: ${networks.ropsten}`,
+    type: 'boolean'
+  })
+  .option('rinkeby', {
+    describe: `Url of the rinkeby ethereum node to connect: ${networks.rinkeby}`,
+    type: 'boolean'
+  })
+  .option('kovan', {
+    describe: `Url of the kovan ethereum node to connect: ${networks.kovan}`,
+    type: 'boolean'
+  })
+  .option('sokol', {
+    describe: `Url of the sokol ethereum node to connect: ${networks.sokol}`,
+    type: 'boolean'
   })
+  .option('poa', {
+    describe: `Url of the poa ethereum node to connect:  ${networks.poa}`,
+    type: 'boolean'
+  })
+  .option('local', {
+    describe: `Url of the local ethereum node to connect:  ${networks.local}`,
+    type: 'boolean'
+  })
+  .check(function(argv) {
+    // Mantain the amount of networks in argv
+    let networksInArgv = 0
```

Comment:
If we remove that validation, we will allow a command to support the duplicity of parameters, example:

`$ eth repl --ropsten --mainnet`

In that case it will take the last parameter. Something to take into account.
There is a boolean variable called 'optionUrlIsNotDefault ' that could be used to check if the url is already set or not. Dou you agree?